src/nbl/video/IOpenGL_PhysicalDeviceBase.h

```diff
@@ -484,6 +484,12 @@ class IOpenGL_PhysicalDeviceBase : public IPhysicalDevice
 
 	bool isSwapchainSupported() const override { return true; }
 
+	SFormatProperties getFormatProperties(asset::E_FORMAT format) const override
+	{
+		_NBL_TODO();
+		return {};
+	}
```

Comment:
@achalpandeyy ask @Crisspl in his infinite wisdom of OpenGL format support

Reply:
Most of what you need for OpenGL
https://github.com/Devsh-Graphics-Programming/Nabla/blob/master/source/Nabla/COpenGLDriver.h#L241

Reply:
@achalpandeyy status? Need info for #250

Reply:
Unimplemented for the GL backend

Reply:
TODO: @Erfan-Ahmadi

Reply:
I'll do it eventually #385